easybuild/easyconfigs/c/Cereal/Cereal-1.3.0.eb

```diff
@@ -0,0 +1,30 @@
+##
+# @Authors:: Maxime Boissonneault, Compute Canada
+# @date: October 15, 2020
+##
+
+easyblock = 'CMakeMake'
+
+name = 'Cereal'
```

Comment:
@mboisson Shouldn't be `cereal`? That's how it's spelled pretty consistently at https://uscilab.github.io/cereal/ (see also `description` below)